fastlane_bot/events/async_event_update_utils.py

```diff
@@ -207,65 +189,9 @@ def get_pool_info(
     return pool_info
 
 
-def sanitize_token_symbol(token_symbol: str, token_address: str, read_only: bool) -> str:
```

Comment:
why are those functions no longer needed?

Reply:
I have no idea, but I've searched through the code, and they do not appear to be called.
I will appreciate your validation of this, of course.